README.md

```diff
@@ -38,7 +38,13 @@ $ git hist
 
 ![Screenshot](http://i.imgur.com/D5AqNUz.png)
 
-*If your terminal displays weird characters, please make sure you're running the latest version of `less` before reporting an issue. Thanks! :beers:*
+*If your terminal displays weird characters, please make sure you're running at least version `471` of `less` before reporting an issue. Thanks! :beers:*
+
+*You can install `less` via [Homebrew](http://brew.sh):*
```

Comment:
Could you make this change, please? In the meantime, I am giving you push access to this repo so that you can merge it as soon as you're done with it :wink: